tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/passes/ConfigurationPassImpl.scala

```diff
@@ -38,7 +43,47 @@ class ConfigurationPassImpl @Inject()(val options: PassOptions,
     * @return true, if the pass was successful
     */
   override def execute(): Boolean = {
-    val module = tlaModule.get
+    var module = tlaModule.get
+
+    // read TLC config if present
+    val filename =
+      options.getOrError("parser", "filename").asInstanceOf[String]
+      .replaceFirst("\\.tla$", "\\.cfg")
+    val basename = FilenameUtils.getName(filename)
+    try {
+      val config = TlcConfigParser.apply(new FileReader(filename))
+      module = new TlcConfigImporter(config, new IdleTracker())(module)
```

Comment:
I don't like that the config file is read automatically if it is present in the file system. Let's add an option `--config <filename>`, similar to TLC. We should always give the user a way to override default options.

Reply:
FWIW: TLC reads a config file `A.cfg` automatically if `A.cfg` is present in the same folder that `A.tla`; the spec that the user check.  In other words, if the config file has a matching name, one does not have to pass `--config ...`.

Reply:
Wow! I did not know that :-)